File: app/src/main/java/com/u1fukui/bbs/model/Category.java
package com.u1fukui.bbs.model;

import java.io.Serializable;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class Category implements Serializable {

    private static final long serialVersionUID = 4312405835923148571L;

    public int id;

    public String name;
}

File: app/src/main/java/com/u1fukui/bbs/model/Category.kt
package com.u1fukui.bbs.model

import se.ansman.kotshi.JsonSerializable
import java.io.Serializable

@JsonSerializable
data class Category(
        val id: Int,
        val name: String
) : Serializable {

    companion object {

        private const val serialVersionUID = 4312405835923148571L
    }
}